src/common/variables.c

```diff
@@ -513,7 +513,7 @@ static char *variable_get_value(dt_variables_params_t *params, char **variable)
           if(length >= 0)
             end = g_utf8_offset_to_pointer(start, MIN(length, start_utf8_length));
           else
-            end = g_utf8_offset_to_pointer(base_value + base_value_length, MAX(length, -1 * start_utf8_length));
```

Comment:
The result of the multiplication of -1 by the unsigned `start_utf8_length` and `base_value_utf8_length` yields an unsigned integer. Signed variables must be used here.

Reply:
Right!